Files/Helpers/StorageItemHelpers.cs

```diff
@@ -16,34 +19,41 @@ public static async Task<IStorageItem> ToStorageItem(this IStorageItemWithPath i
 
         public static async Task<TOut> ToStorageItem<TOut>(string path, IShellPage associatedInstance = null) where TOut : IStorageItem
         {
-            FilesystemResult<StorageFile> file = null;
-            FilesystemResult<StorageFolder> folder = null;
-
-            if (associatedInstance == null)
+            try
```

Comment:
Is this try/catch necessary? `FilesystemTasks.Wrap` and FilesystemViewModel.Get*** can't throw exceptions